src/app/PageWrapper.tsx

```diff
@@ -40,37 +40,45 @@ export const PageWrapper: FC = ({ children }) => {
               : undefined,
         }}
       >
-        <Header fullWidth={true} />
-        <Flex
-          display={['block', 'block', 'none', 'none']}
-          p="8px"
-          mx="auto"
-          width="100%"
-          flexDirection={'column'}
-          px={'16px'}
-        >
-          <SearchComponent variant="small" mr="16px" width="100%" maxWidth="760px" />
-        </Flex>
-        <Flex flexDirection="column" width="100%" minHeight="100%" position="relative" flexGrow={1}>
-          {SITE_NOTICE_ENABLED && (
-            <Box px="20px">
-              <Notice label={SITE_NOTICE_BANNER_LABEL} message={SITE_NOTICE_BANNER_MESSAGE} />
-            </Box>
-          )}
+        <Box marginLeft={`calc(100vw - 100%)`}>
+          <Header />
           <Flex
-            as="main"
+            display={['block', 'block', 'none', 'none']}
+            p="8px"
             mx="auto"
             width="100%"
-            flexGrow={1}
-            height="100%"
-            maxWidth={'1280px'}
+            flexDirection={'column'}
+            px={'16px'}
+          >
+            <SearchComponent variant="small" mr="16px" width="100%" maxWidth="760px" />
+          </Flex>
+          <Flex
             flexDirection="column"
-            px={['16px', '16px', '32px']}
+            width="100%"
+            minHeight="100%"
+            position="relative"
+            flexGrow={1}
           >
-            {children}
+            {SITE_NOTICE_ENABLED && (
+              <Box px="20px">
+                <Notice label={SITE_NOTICE_BANNER_LABEL} message={SITE_NOTICE_BANNER_MESSAGE} />
+              </Box>
+            )}
+            <Flex
+              as="main"
+              mx="auto"
+              width="100%"
+              flexGrow={1}
+              height="100%"
+              maxWidth={'1280px'}
```

Comment:
It seems like this value should be a shared const somewhere rather than hard-coded